protocol/state.re

```diff
@@ -13,6 +13,7 @@ type t = {
   // consensus
   validators: Validators.t,
   validators_hash: BLAKE2B.t,
+  last_seen_membership_change_timestamp: float,
```

Comment:
That's not right, this is part of the local consensus data, together with `last_state_root_update` and `last_applied_block_timestamp`

Reply:
I don't understand this comment - are you saying `last_seen_membership_change_timestamp` should be moved to node local state? `last_state_root_update` and `last_applied_block_timestamp` are currently a part of protocol state and not node - are you saying they should be moved too?

Reply:
Thanks for explaining offline. Making the changes in the next commit